data/docs/frontend-monitoring/sending-logs.mdx

```diff
@@ -1,8 +1,9 @@
 ---
 date: 2025-08-17
-tags : [SigNoz Cloud, Self-Host]
+tags: [SigNoz Cloud, Self-Host, Frontend, JavaScript]
```

Comment:
Frontend and Js tags don't exist. Only SigNoz Cloud and Self-Host are valid tags